modules/e2ee/E2EEncryption.js

```diff
@@ -64,13 +122,47 @@ export class E2EEncryption {
     }
 
     /**
-     * Sets the key to be used for End-To-End encryption.
-     *
-     * @param {string} key - the key to be used.
-     * @returns {void}
+     * TODO.
+     */
+    _onParticipantJoined(id) {
+        logger.debug(`Participant ${id} joined`);
+
+        if (this._enabled) {
+            this._rotateKey();
```

Comment:
can you call a .ratchet which for now calls rotateKey?

Reply:
👍